src/main/webapp/app/markdown-editor/markdown-editor.component.ts

```diff
@@ -156,14 +166,12 @@ export class MarkdownEditorComponent implements AfterViewInit, OnInit {
      * @function parseLineForSpecialCommand
      * @desc check which specialCommand is contained within the textLine and remove the specialCommand from the textLine
      * @param textLine {string} from the parse function
-     * @return array of the textLine with the corresponding specialCommand
+     * @return array of the textLine with the previously contained specialCommand
```

Comment:
what does 'previously contained' mean?
Also I think we wanted to rename specialCommand into domainCommand --> please use the term domainCommand consistently!!!